### ISSUE TITLE
Can you please share a quick-start on how to simply run the code in VSC?

### ISSUE THREAD

**Issue by MatanYamin** [jimm98y/SharpAndroidTV#4]
Hi, when trying to run the code, I get an error when trying "await tvPairingClient.InitiatePairingAsync();", I get:
Interop+AppleCrypto+SslException: handshake failure

**Comment by jimm98y** [comment]
I don't know when I get to creating the quickstart, I do not have the TV anymore. This was tested on Windows 11 for ARM64 on NET6 (running in Parallels on M1). NET6/NET8 have different underlying implementations of the TLS on Windows and other OSs, where on Windows they are using Windows APIs, but on other OSs (Linux/MacOS) they rely upon OpenSSL. So your issue looks like it might be related to the TLS implementation - some recent TVs started requiring TLS1.3. For that to work you have to use the DLL in NET8 application and to explicitly set TLS to 1.3 through the ctor - netstandard2.0 does not support TLS1.3 so this workaround only works on NET8/NET Framework 4.8. If it won't work, a more cross-platform implementation would require implementing the TLS1.3 on top of TcpClient using BouncyCastle, bypassing the underlying platform-specific implementation.
As for an example, basically copy/pasting the code in the same order in which it appears in the readme into a sample console app should give you what you are looking for.

**Comment by MatanYamin** [comment]
Well, I actually managed to build it my self with C# .

My only issue is - after pairing successfully, I can't create SSL connection to port 6466 for the command sending.

Do I need to open SSL connection with authentication for that part using port 6466? Because I get Authentication failed on that part (your answer will help me a lot).

I think i'm doing something wrong there.

**Comment by jimm98y** [comment]
Yes, to connect on port 6466 you need to use the certificate from the pairing step and use it as a client certificate for the connection.
